exploration/number-selection.md

```diff
@@ -92,44 +95,274 @@ ICU MF1 messages using `plural` and `selectordinal` should be representable in M
+   -  `exact`
+- `compactDisplay`
+   - `short` (default)
+   - `long`
+- `currency`
+   - valid [Unicode Currency Identifier](https://cldr-smoke.unicode.org/spec/main/ldml/tr35.html#UnicodeCurrencyIdentifier)
+     (no default)
+- `currencyDisplay`
+   - `symbol` (default)
+   - `narrowSymbol`
+   - `code`
+   - `name`
+- `currencySign`
+  - `accounting`
+  - `standard` (default)
+- `notation`
+   - `standard` (default)
+   - `scientific`
+   - `engineering`
+   -  `compact`
+- `numberingSystem`
+   - valid [Unicode Number System Identifier](https://cldr-smoke.unicode.org/spec/main/ldml/tr35.html#UnicodeNumberSystemIdentifier)
+     (default is locale-specific)
+- `signDisplay`
+   -  `auto` (default)
+   -  `always`
+   -  `exceptZero`
+   -  `never`
+- `style`
+  - `decimal` (default)
+  - `currency`
+  - `percent`
+  - `unit`
+- `unit`
+   - (anything not empty)
+- `unitDisplay`
+   - `long`
+   - `short` (default)
+   - `narrow`
+- `minimumIntegerDigits`
+  - (positive integer, default: `1`)
+- `minimumFractionDigits`
+  - (positive integer)
+- `maximumFractionDigits`
+  - (positive integer)
+- `minimumSignificantDigits`
+  - (positive integer)
+- `maximumSignificantDigits`
+  - (positive integer)
+
+
+> [!NOTE]
+> We have a requirement for `offset` but have not provided for it yet??
+
+The value `plural` is default for the option `select` 
+because it is the most common use case for numeric selection.
+It can be used for exact value matches but also allows for the grammatical needs of other 
+languages using CLDR's plural rules.
+This might not be noticeable in the source language (particularly English), 
 but can cause problems in target locales that the original developer is not considering.
 
 > For example, a naive developer might use a special message for the value `1` without
 > considering other locale's need for a `one` plural:
 >
 > ```
 > .match {$var}
-> [1] {{You have one last chance}}
-> [one] {{You have {$var} chance remaining}} // needed by languages such as Polish or Russian
-> [*] {{You have {$var} chances remaining}}
+> 1   {{You have one last chance}}
+> one {{You have {$var} chance remaining}} // needed by languages such as Polish or Russian
+>                                          // such locales typically require other keywords
+>                                          // such as two, few, many, and so forth
+> *   {{You have {$var} chances remaining}}
 > ```
 
-Additional options such as `minimumFractionDigits` and others already supported by `:number`
-should also be supported.
+### Selection
+
+When implementing [`MatchSelectorKeys`](spec/formatting.md#resolve-preferences), 
+numeric selectors perform as described below.
+
+- Let `return_value` be a new empty list of strings.
+- Let `operand` be the resolved value of the _operand_.
+  If the `operand` is not a number type, emit a _Selection Error_
+  and return `return_value`.
+  > This might result in the fallback message (logo) being rendered.
```

Comment:
This is not the case according to our current spec where an all-`*` variant is required.
```suggestion
```

Reply:
Although if the operand isn't a number now, it probably won't format later. But that could result in a more usable fallback message I suppose.